mautic/mautic
# WIP: Enhanced user status information

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ✖️ ]
| New feature/enhancement? (use the a.x branch)      | [ ✔️ ]
| Deprecations?                          | [✖️]
| BC breaks? (use the c.x branch)        | [✖️]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


#### Description:

This PR itroduced better visibility of the Mautic users status:
* Show "Last Login" and "Publication status" in the Users list table
* Show "Recent Acivity " in the User detail view

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to "Users" list view - you should see the new columns  "Last Login" and "Publication status"
3. When you create a new user, log in, deactivate, ... that show up here
4. When you open one users details, you should see recent activities on the right hand side.
